aks/entrypoint.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 set -e
-export AZURE_HTTP_USER_AGENT="GITHUBACTIONS_${GITHUB_ACTION}_${GITHUB_REPOSITORY}"
+export AZURE_HTTP_USER_AGENT="GITHUBACTIONS_${GITHUB_ACTION_NAME}_${GITHUB_REPOSITORY}"
```

Comment:
what's the benefit of using action name as is v/s descriptive name ? Just for my understanding, why are we making this change?

Reply:
we were passing GitHub action which was "custom name provided by user", but we need to pass actual GitHub action (AZ CLI / AZ Web App deploy) for telemetry